builtins/amp-video.md

```diff
@@ -16,13 +16,51 @@ limitations under the License.
 
 ### <a name="amp-video"></a> `amp-video`
 
-A replacement for the HTML5 `video` tag. Like all embedded external resources in a AMP file, the video is lazily loaded only when the `amp-video` element is in or near the viewport.
-
-The `amp-video` component is only to be used for direct HTML5 video file embeds.
+<table>
+   <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>A replacement for the HTML5 <code>video</code> tag; only to be used for direct HTML5 video file embeds.</td>
+  </tr>
+   <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Stable</td>
+  </tr>
+   <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><a href="https://github.com/ampproject/amphtml/blob/master/examples/everything.amp.html">everything.amp.html</a></td>
+  </tr>
+</table>
+
+The following lists validation errors specific to the `amp-video` tag
+(see also `amp-video` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii):
+
+<table>
+  <tr>
+    <th width="40%"><strong>Validation Error</strong></th>
```

Comment:
spec should be fixed now. https://github.com/ampproject/amphtml/blob/master/extensions/amp-carousel/amp-carousel.md wording might a be a bit unclear, might need help on that @Meggin